### ISSUE TITLE
zlib required on install

### ISSUE THREAD

**Issue by bioSandMan** [dstreett/Super-Deduper#19]
Not sure if this is an "issue" or a "need to know". The library zlib is required for install. I installed zlib using

$sudo apt-get install zlib1g-dev

I'm on ubuntu 16.04 LTS. Without super user access, one may need to download and install the package manually. I would recommend against including any third party external library files as part of the download since that would require maintaining the latest copy to protect against security flaws. What I mean is, we shouldn't jump to include the header files in our distros since we aren't maintainers of those external libraries.

**Comment by dstreett** [comment]
Correct, for the current master zlib is required. However, the develop branch (post huge code refractor) does not need zlib.

I will keep this open until we decide to merge develop into master. The build testing should be occurring on develop and not on current master.
